aws/vars.tf

```diff
@@ -9,6 +9,12 @@ variable "nodes" {
   default     = "3"
 }
 
+variable "ha" {
+  description = "Whether to use placement groups to create an HA topology"
+  type        = bool
+  default     = false
+}
+
```

Comment:
Can you update the README of `aws/` with this variable?

This can be done in a follow-up/v2 branch since I think the Readme is outdated.

Reply:
Went ahead and included this in the most recent commit.